### ISSUE TITLE
EnumerateStereoisomers gives index error with imine

### ISSUE THREAD

**Issue by HellevdM** [rdkit/rdkit#7707]
**Describe the bug**
StereoEnumeration gives the range error below with molecules containing a imine.
It seems to give this error with any molecule matching `*[#6](=N)*`,
like `COc1c2oc(=N)c(C(=O)NC3CCCC3)cc2ccc1`

```
RuntimeError: Range Error
	idx2
	Violation occurred on line 352 in file Code/GraphMol/ROMol.cpp
	Failed Expression: 4294967295 < 4
	RDKIT: 2024.03.4
	BOOST: 1_85
```

**To Reproduce**
```from rdkit import Chem
from rdkit.Chem.EnumerateStereoisomers import EnumerateStereoisomers, StereoEnumerationOptions

mol = Chem.MolFromSmiles("CC(=N)C")

options = StereoEnumerationOptions(onlyUnassigned=False, unique=True)
isomers = list(EnumerateStereoisomers(mol, options))
```

**Screenshots**
![image](https://github.com/user-attachments/assets/af9eb2ad-3162-4e63-9823-ec1c5e5aea57)

**Configuration:**
 - RDKit version: 2024.03.5 & 2024.03.4 (2023.03.3 does not have the issue)
 - OS: Rocky Linux 8:10
 - Python version (if relevant): 3.10
 - Are you using conda? yes
 - If you are using conda, which channel did you install the rdkit from? pypi

**Comment by greglandrum** [comment]
I believe this is a duplicate with https://github.com/rdkit/rdkit/issues/7608
It is fixed and will be out in the 2024.03.6 release.

**Comment by LucaCappelletti94** [comment]
When will `2024.03.6` be released? I have just encountered this issue myself. For reference, this is the code raising the exception:

```python
from rdkit.Chem.EnumerateStereoisomers import (
    EnumerateStereoisomers,
    StereoEnumerationOptions,
)
from rdkit.Chem import Mol, MolFromSmiles

smile = "CN(CCC(N)CC(=O)NC1C=CC(n2ccc(=O)[nH]c2=O)OC1C(=O)O)C(=N)N"
mol: Mol = MolFromSmiles(smile) 

stereoisomers = list(EnumerateStereoisomers(mol))
```
